switchmap/server/db/table/vlanport.py

```diff
@@ -123,6 +123,40 @@ def insert_row(rows):
         db.db_add_all(1185, inserts)
 
 
+def bulk_insert_rows(rows):
```

Comment:
1. Remove.
2. Update `db.add_all` to use the new `bulk_insert_rows` method
3. Apply this to all the` switchmap/server/db/table` files. It's simpler. There v2.0 of sqlalchemy has a `bulk_insert_rows` replacement.